manifests/weave/base/validator.yaml

```diff
@@ -147,7 +147,7 @@ spec:
           name: tm-initialization
           subPath: genesis.json
           readOnly: true
-        - mountPath: /etc/tendermint/config.toml
+        - mountPath: /data/tendermint/config/config.toml
```

Comment:
🤔 maybe init ignores the config completely...
L146 also points to /etc/tendermint...

Reply:
yeah it does, but then we can mount it on top.